packages/compass-schema/src/components/export-schema/export-schema-modal.tsx

```diff
@@ -0,0 +1,321 @@
+import {
+  Banner,
+  BannerVariant,
+  Checkbox,
+  css,
+  InfoModal,
+  Option,
+  Select,
+  spacing,
+  TextInput,
+  WorkspaceContainer,
+} from '@mongodb-js/compass-components';
+import type { Schema } from 'mongodb-schema';
+import React, { useMemo, useState, useCallback } from 'react';
+
+import { JSONEditor } from '@mongodb-js/compass-editor';
```

Comment:
This needs to be declared as a dependency in package.json :)

Reply:
true